src/bin/DevChainOptions.ts

```diff
@@ -0,0 +1,25 @@
+export default class DevChainOptions {
+  public static isDevChain(chain: string, options = { origin: '' }): boolean {
+    return (
+      chain === 'origin'
+      || chain === 'dev'
+      || chain === 'auxiliary'
+      || options.origin === 'dev'
+    );
+  }
+
+  public static getDevChainParams(chain: string, options = { origin: '' }) {
+    let chainInput = chain;
+    const optionInput = Object.assign({}, options);
+    if (chain === 'origin') {
+      chainInput = 'dev';
+    } else if (chain === 'auxiliary') {
+      chainInput = '1000';
```

Comment:
We should avoid this hardcoding. It can be constant variable which can be used across files.